### PR TITLE
Feature/devsu 1551 investigate print numbering

### PR DESCRIPTION
**Pull request by Anna-log7**
This PR adds the library `paged-js` to implement some media queries that aren't available in browsers. The main changes here are that `paged-js` now adds a page count to the bottom right of a print report, and a header to the top of every page with some info.

### CODE REVIEW

**Comment by elewis2** [comment]
Currently the box that the analyst comments are printed in is open-ended at the bottom of every page it's on except its last. I don't think this is related to this ticket, but has this been addressed elsewhere? Is it the result of a decision?

**Comment by Anna-log7** [comment]
> 
> 
> Currently the box that the analyst comments are printed in is open-ended at the bottom of every page it's on except its last. I don't think this is related to this ticket, but has this been addressed elsewhere? Is it the result of a decision?

It's a limitation of content spanning multiple pages when printing. That element is as long as the comments are so the pages break it up and as far as I know there's no way to close those borders since there's no way to fetch where a page break in dynamic generated content is happening

**Review by elewis2 — APPROVED** [review]

**Review by kttkjl — APPROVED** [review]

File: app/index.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  .running-right {
+    position: running(run-right);
+  }
+
+  .page-break {
+    break-before: page;
+  }
+
+  @page {
```
[the remark this answers is not in the document]
Comment (Anna-log7) [reply app/index.css:18]: It only works with paged.js unfortunately. There's been a CSS3 proposal to support this that's been open for years with no movement whatsoever.

**Comment by kttkjl** [comment]
@elewis2 's comment on the box is quite a tough thing to tackle ... some thoughts

are the borders a requirement in terms of printing the page?  If not then can we get away with just removing it entirely?

**Comment by kttkjl** [comment]
image
Oh actually I hope this is just my problem, but it seems like page content is being cut ...?

**Comment by Anna-log7** [comment]
> 
> 
> @elewis2 's comment on the box is quite a tough thing to tackle ... some thoughts
> 
> are the borders a requirement in terms of printing the page? If not then can we get away with just removing it entirely?

As far I know, no

> 
> 
> image
> Oh actually I hope this is just my problem, but it seems like page content is being cut ...?

Oh I had this at one point so it's probably not just you. I'll check it out.

**Comment by Anna-log7** [comment]
> 
> 
> image
> Oh actually I hope this is just my problem, but it seems like page content is being cut ...?

I think this is fixed now. Can you try it out? I also ended up removing the box as it was a pain to deal with

**Comment by kttkjl** [comment]
image

Confirmed working

**Review by kttkjl — APPROVED** [review]

**Review by elewis2 — APPROVED** [review]

**Pull request merged**